problems/lib/problem2.rb

```diff
@@ -0,0 +1,26 @@
+class Problem2
+  def get_largest_two_numbers(array)

```

Comment:
The first part when initializing max1 and max2 variables can be refactored as per my previous [reply](https://github.com/Agilefreaks/Test/pull/18/files/6994d074a8a8d329459c390cee38354ed514af65#r127580557) to @pirvudoru.

The block of code in the loop can be refactored as such:
1. `max1, max2 = num > max1 ? [num, max1] : [max1, num]`
Although not sure this approach would be acceptable, since it basically would be possible to do one additional unrequited assignment of max1 in max1, max2 = [max1, num], compared to the current code. On top of that seems that [Ruby coding style](https://github.com/bbatsov/ruby-style-guide) advises against this kind of usage for parallel assignment.